backend/tests/endpoints/test_room.py

```diff
@@ -1,4 +1,286 @@
+from unittest import mock
+
+import pytest
 from fastapi.testclient import TestClient
 from main import app
+from utils.db import DataStorage
 
 client = TestClient(app)
+
+join_room_test_url = (
+    "api/v1/org/3467sd4671a5f5478df56u911/rooms/23dg67l0eba8adb50ca13a24/"
+    + "members/619baa5939d386c1a5f54782"
+)
+
+join_room_test_payload = {
+    "619baa5939d386c1a5f54782": {
+        "closed": False,
+        "role": "member",
+        "starred": False,
+    }
+}
+
+fake_room_data = {
+    "room_name": "General",
+    "room_type": "CHANNEL",
+    "room_members": {
+        "61696f5ac4133ddaa309dcfe": {
+            "closed": False,
+            "role": "admin",
+            "starred": False,
+        },
+        "6169704bc4133ddaa309dd07": {
+            "closed": False,
+            "role": "admin",
+            "starred": False,
+        },
+        "619baa5c1a5f54782939d386": {
+            "closed": False,
+            "role": "member",
+            "starred": False,
+        },
+    },
+    "created_at": "2022-01-11 03:18:02.364291",
+    "description": None,
+    "topic": "General Information",
+    "is_private": False,
+    "is_archived": False,
+    "id": "23dg67l0eba8adb50ca13a24",
+    "org_id": "3467sd4671a5f5478df56u911",
+    "created_by": "619ba4671a5f54782939d385",
+}
+
+success_response = {
```

Comment:
just noticed this is named success_response. potential conflicts with other codes.

Reply:
@mykie88

Reply:
I don't understand how @pauline-banye

Reply:
This success response is particular to only the join room test. It's not re-useable for other tests the way the fake_room_data is and it's not explicitly named.

Reply:
oh I get now